MAINTAINERS.yml

```diff
@@ -3361,6 +3361,7 @@ NXP Platforms (MCU):
     - soc/nxp/mcx/
     - dts/arm/nxp/
     - samples/boards/nxp*/
+    - boards/phytec/phycore_rt*/
```

Comment:
@dleach02 are you okay with putting this board here? We have no maintainer for our boards and IMO it makes sense to work closely together with you to always stay in sync.

Reply:
I think you need to setup a phytec area in the maintainer file and assign someone from Phytec to be responsible for maintaining. NXP can't assume maintainership responsibility for all 3rd party boards that use our SOC. We don't mind helping but we don't have the internal bandwidth to obtain and add the board to our HW test range.

Reply:
We got `PHYTEC platforms` setup.

Btw we're also adding our i.MX93 SOM right now...